lib/fastly_nsq/consumer.rb

```diff
@@ -11,13 +11,13 @@ class Consumer
     def initialize(topic:, channel:, ssl_context: nil, connector: nil)
       @topic       = topic
       @channel     = channel
-      @ssl_context = SSLContext.new(ssl_context)
+      @tls_options = TlsOptions.new(ssl_context).to_h
```

Comment:
I don't think we use the passed params anywhere ATM... we lean on the `ENV` variables as defaults so changing that shouldn't make downstream work much harder at all...